GraphQL-TSC.md

```diff
@@ -139,15 +139,15 @@ After November 1st, the TSC will collect self-nominations from the community. Af
 
 #### Initial period
 
-To provide consistency TSC elections will begin in 2021 for the 2022 term. Prior to the first election, half of the existing members will be selected via coin toss to serve an initial one-year term.
+To provide consistency TSC elections will begin in 2021 for the 2022 term. Prior to the first election, half of the existing members will be selected via ["coin toss"](https://gist.github.com/leebyron/cbda7f0c604915d9200af5626b1fcf1b) to serve an initial one-year term.
```

Comment:
if we want to reuse this code it should not to hosted somewhere outside of this repo, but committed here.

Reply:
I think it's more of a "here's a stable random algorithm to show everything was fair" evidence kind of thing, rather than "here's a tool to use again."

Reply:
Benjie is correct here - I just wanted to illustrate that I deferred the decision making process to documented repeatable deterministic math instead of undocumented unrepeatable coin-flipping.

This is a one-time thing, so there's no reason to repeat or reuse this code, so checking it in might be low SNR

Reply:
I would recommend then to make this more clear. But I don't consider this a blocker.

Reply:
Good suggestion. Let me add a small clarifying note before I merge